yolact.py

```diff
@@ -470,6 +131,16 @@ def __init__(self):
         self.detect = Detect(cfg.num_classes, bkg_label=0, top_k=cfg.nms_top_k,
             conf_thresh=cfg.nms_conf_thresh, nms_thresh=cfg.nms_thresh)
 
+
+        # set default backbone weights
+        self.init_weights(backbone_path='weights/' + cfg.backbone.path)
+
+        # GPU
+        #TODO try half: net = net.half()
+        self.cuda()
+        torch.set_default_tensor_type('torch.cuda.FloatTensor')
```

Comment:
If this didn't break anything, then it's fine, but this should probably go at the start of __init__ not here.